lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/README.md

```diff
@@ -30,8 +30,8 @@ The normalized (aka "probabilist") [Hermite polynomials][hermite-polynomial] are
 
 <!-- <equation class="equation" label="eq:normalized_hermite_polynomials" align="center" raw="TODO" alt="Equation for normalized Hermite polynomials."> -->
```

Comment:
@jrreed83 This is the element you need to change. Basically, we have logic which reads the comment and can autogenerate the HTML which follows. I did not provide you the command to do that. Sorry about that. For now, just copy the equation to the `raw=` attribute in the comment at L31. This is why it is marked with a `TODO`.

Reply:
Ok, I thought you meant the HTML snippet containing the image.  My bad.

Reply:
Yes, this was my fault. I left out another command, which I typically don't run because I habitually just manually copy the TeX from the comment attribute to the HTML element attribute to ensure that they are the same. We do, however, have tooling that automates all of this. I just have not documented it, yet. :|

Reply:
That's all really good stuff.  How long did it take the core team to come up with all the tooling?  Did it evolve naturally, or did you know what you wanted automated when you started?

Reply:
I just checked the Hermite equation at the URL the tooling generated.  It looks like the wikipedia article equation.

Reply:
@jrreed83 Awesome. Thanks for doing this. Not sure how your previewed the equation, but one tool I use is the site https://www.codecogs.com/eqnedit.php, which allows me to render an equation in the browser without needing to actually compile TeX myself. Once I am satisfied with the equation, I copy and paste it in to a README.

Re: tooling. By-and-large, the tooling has evolved naturally. As the project grows, we encounter new issues or realize new aspects of the dev process that we want to automate, and, once realized, we build the tooling. Still a considerable amount of tooling still to be written!